stage1_fly/run/main.go

```diff
@@ -333,8 +338,8 @@ func stage1() int {
 		return 1
 	}
 
-	if err := os.Chdir("/"); err != nil {
-		log.PrintE("can't change to root new directory", err)
+	if err := os.Chdir(workDir); err != nil {
```

Comment:
I'm also struggling to see the escape vector here. AIUI escaping a chroot requires creating a new directory and chrooting into it before repeatedly cding up, e.g. http://www.bpfh.net/simes/computing/chroot-break.html